tests/test_build_datapackage.py

```diff
@@ -13,12 +14,42 @@
 path_default = PATH_TEST_FILES / "_files"
 
 
+def test_refactor_timeseries():
+    timeseries = pd.DataFrame(
+        {
+            "timeindex_start": ["01:00:00", "01:00:00", "09:00:00"],
+            "timeindex_stop": ["03:00:00", "03:00:00", "10:00:00"],
+            "timeindex_resolution": ["P0DT01H00M00S", "P0DT01H00M00S", "P0DT01H00M00S"],
+            "region": ["BB", "HH", "HH"],
+            "onshore": [[1, 2, 3], [4, 5, 6], [33, 34]],
+            "offshore": [[7, 8, 9], [10, 11, 12], [35, 36]],
+        }
+    )
+
+    refactored_ts = refactor_timeseries(timeseries)
```

Comment:
This function is missing. I cant find it...

Reply:
TS refactoring has been moved to `data_adapter`. Seems to be a legacy test